core/src/connection/pool.rs

```diff
@@ -1058,6 +1062,13 @@ impl ConnectionLimits {
         self
     }
 
+    /// Configures the maximum number of concurrent total established connections (inbound or
+    /// outbound).
+    pub fn with_max_established_total(mut self, limit: Option<u32>) -> Self {
```

Comment:
```suggestion
    /// Configures the maximum number of concurrent established connections (both inbound and
    /// outbound).
    pub fn with_max_established(mut self, limit: Option<u32>) -> Self {
```

I would find this naming more intuitive, given that `total` is relative and requires the context of `inbound` and `outbound`.